homeassistant/components/axis.py

```diff
@@ -202,69 +209,71 @@ def vapix_service(call):
         """Service to send a message."""
         for _, device in AXIS_DEVICES.items():
             if device.name == call.data[CONF_NAME]:
-                response = device.do_request(call.data[SERVICE_CGI],
-                                             call.data[SERVICE_ACTION],
-                                             call.data[SERVICE_PARAM])
+                response = device.vapix.do_request(call.data[SERVICE_CGI],
```

Comment:
The service method is not annotated, so assuming this call is to a sync method ?

Reply:
At the moment it is a wrapper for the request library.
https://github.com/Kane610/axis/blob/7c143aebe2b03c3b7d2c2722dea947f27efe1dfa/axis/vapix.py#L40